Native/Text/src/ARText.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { createElement, useEffect, useRef, useState } from "react";
+import { ARTextProps } from "../typings/ARTextProps";
+import { Style } from "mendix";
+import { MeshComponent, setAttributes } from "../../../Shared/ComponentParent/src/MeshComponent";
+import { useGizmoComponent } from "../../../Shared/ComponentParent/src/useGizmoComponent";
+import { Mesh, MeshBuilder, Scene, Texture, Vector3 } from "@babylonjs/core";
+import { retrieveImageFromNumber } from "../../../Shared/ComponentParent/src/retrieveImageFromNumber";
+import font from "../../../Shared/ComponentParent/utils/Open Sans_Regular.json";
+import earcut from "earcut";
+
+export function ARText(props: ARTextProps<Style>): React.ReactElement {
+    (window as any).earcut = earcut;
```

Comment:
Why you need store `earcut` it as global?

Reply:
Otherwise it doesn't get included in the bundle, but let me double-check that.

Reply:
Yeah I double checked this, I can't inject it the way I'm using it. https://forum.babylonjs.com/t/using-next-js-got-earcut-is-not-defined-error/35940/5 here's it suggested by the babylon guys themselves.